### PR TITLE
only load pdf files

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes   https://github.com/owncloud/web/issues/9309

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-pdf-viewer/src/App.vue [hunk 1]
```diff
@@ -51,7 +51,13 @@ export default defineComponent({
     async loadPdf(fileContext) {
       try {
         this.loading = true
-        this.resource = await this.getFileInfo(fileContext)
+        const resource = await this.getFileInfo(fileContext)
+
+        if (resource.mimeType !== 'application/pdf') {
+          return
```
Comment (JanAckermann) [inline packages/web-app-pdf-viewer/src/App.vue:57]: note that return in try...catch...finally... block will call the finally block and sets loading to false

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/36984/45/1
:boom: The oC10XGAPortrait1 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/36984/46/1
:boom: The oC10XGAPortrait2 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/36984/44/1
:boom: The oC10SharingExternalRoot tests pipeline failed. The build has been cancelled.

**Review by lookacat — APPROVED** [review]

**Pull request merged**